.github/workflows/build-and-deploy.yml

```
@@ -92,16 +92,12 @@ jobs:
       run: docker image push --all-tags ${{ env.DOCKER_IMAGE }}
 
     - name: Run Snyk to check Docker image for vulnerabilities
-      uses: Wandalen/wretry.action@v1.3.0
+      uses: snyk/actions/docker@master
       env:
         SNYK_TOKEN: ${{ steps.get-secret.outputs.snyk_token }}
       with:
-        attempt_limit: 3
-        attempt_delay: 2000
-        action: snyk/actions/docker@master
-        with: |
-          image: ${{ env.DOCKER_IMAGE }}:${{ env.IMAGE_TAG }}
-          args: --file=Dockerfile --severity-threshold=high --exclude-app-vulns
+        image: ${{ env.DOCKER_IMAGE }}:${{ env.DOCKER_IMAGE_TAG }}
```

It looks like maybe this line has changed from the previous value?
```suggestion
        image: ${{ env.DOCKER_IMAGE }}:${{ env.IMAGE_TAG }}
```